test/acceptance/docker/test/support/prepare_db.sh

```diff
@@ -106,42 +101,9 @@ if test x"$PREPARE_PGSQL" = xyes; then
   curl https://cartodb-libs.global.ssl.fastly.net/carto-vl/assets/pop_density_points.sql -o sql/pop_density_points.sql
   curl https://cartodb-libs.global.ssl.fastly.net/carto-vl/assets/sf_stclines.sql -o sql/sf_stclines.sql
 
-  LOCAL_SQL_SCRIPTS='analysis_catalog windshaft.test cdb_analysis_check cdb_invalidate_varnish mnmappluto monarch_migration_1 pop_density_points sf_stclines '
-  REMOTE_SQL_SCRIPTS='CDB_QueryStatements CDB_QueryTables CDB_CartodbfyTable CDB_TableMetadata CDB_ForeignTable CDB_UserTables CDB_ColumnNames CDB_ZoomFromScale CDB_OverviewsSupport CDB_Overviews CDB_QuantileBins CDB_JenksBins CDB_HeadsTailsBins CDB_EqualIntervalBins CDB_Hexagon CDB_XYZ CDB_EstimateRowCount CDB_RectangleGrid'
-
-  if test x"$DOWNLOAD_SQL_FILES" = xyes; then
-    CURL_ARGS=""
-    for i in ${REMOTE_SQL_SCRIPTS}
-    do
-        CURL_ARGS="${CURL_ARGS}\"https://github.com/CartoDB/cartodb-postgresql/raw/master/scripts-available/$i.sql\" -o sql/$i.sql "
-    done
-    echo "Downloading and updating: ${REMOTE_SQL_SCRIPTS}"
-    echo ${CURL_ARGS} | xargs curl -L -s
-  fi
-
-  psql -c "CREATE EXTENSION IF NOT EXISTS plpythonu;" ${TEST_DB}
-  ALL_SQL_SCRIPTS="${REMOTE_SQL_SCRIPTS} ${LOCAL_SQL_SCRIPTS}"
-  for i in ${ALL_SQL_SCRIPTS}
-  do
-    # Strip PARALLEL labels for PostgreSQL releases before 9.6
-    if [ $PG_PARALLEL -eq 0 ]; then
-        TMPFILE=$(mktemp /tmp/$(basename $0).XXXXXXXX)
-        sed -e 's/PARALLEL \= [A-Z]*,/''/g' \
-            -e 's/PARALLEL [A-Z]*/''/g' sql/$i.sql > $TMPFILE
-        mv $TMPFILE sql/$i.sql
-    fi
-    cat sql/${i}.sql |
-      sed -e 's/cartodb\./public./g' -e "s/''cartodb''/''public''/g" |
-      sed "s/:PUBLICUSER/${PUBLICUSER}/" |
-      sed "s/:PUBLICPASS/${PUBLICPASS}/" |
-      sed "s/:TESTUSER/${TESTUSER}/" |
-      sed "s/:TESTPASS/${TESTPASS}/" |
-      sed "s/cartodb_user_de952f5f-6a86-467d-8c27-72cf6be3395e/${TESTUSER}/" |
-      sed "s/tileuser/${PUBLICUSER}/" |
-      sed "s/TO publicuser/TO ${PUBLICUSER}/" |
-      grep -v ".cdb_checkquota" |
-      PGOPTIONS='--client-min-messages=WARNING' psql -q -v ON_ERROR_STOP=1 ${TEST_DB} > /dev/null || exit 1
-  done
+  psql -c "CREATE EXTENSION IF NOT EXISTS postgis CASCADE;" ${TEST_DB}
+  psql -c "CREATE EXTENSION IF NOT EXISTS cartodb CASCADE;" ${TEST_DB}
+  psql -c "CREATE EXTENSION IF NOT EXISTS plpythonu CASCADE;" ${TEST_DB}
```

Comment:
Not related to the issue, but you don't need to install this individually. See the current Windshaft-config in:

https://github.com/CartoDB/Windshaft-cartodb/blob/master/test/support/prepare_db.sh#L73-L76